src/nvim/api/buffer.c

```diff
@@ -1137,12 +1137,16 @@ Boolean nvim_buf_is_loaded(Buffer buffer)
   return buf && buf->b_ml.ml_mfp != NULL;
 }
 
-/// Deletes the buffer. See |:bwipeout|
+/// Deletes the buffer.
 ///
 /// @param buffer Buffer handle, or 0 for current buffer
 /// @param opts  Optional parameters. Keys:
 ///          - force:  Force deletion and ignore unsaved changes.
-///          - unload: Unloaded only, do not delete. See |:bunload|
+///          - action: (default: "wipeout")
+///             - "wipeout": Equivalent to |:bwipeout|.
+///             - "unload":  Equivalent to |:bunload|.
+///             - "delete":  Equivalent to |:bdelete|.
+///          - unload: [deprecated] please use `action="unload"` instead.
```

Comment:
don't use "please" in docs or messages, it's unnecessary and just takes up space